envs/cheetah_agent.py

```diff
@@ -1,59 +1,123 @@
-from move_agent import MoveTeleportSimulator, ObsIm, BaseRewarder
-
-class HalfCheetahSimulator(MoveTeleportSimulator):
```

Comment:
The Cheetah Simulator can inherit from: https://github.com/pulkitag/rlmaster/blob/master/core/mujoco_environment.py#L60 
An instatiation of BaseSimulator specifically for Mujoco Environments.